Podcast/SearchUsersEndpointRequest.swift

```diff
@@ -18,7 +18,15 @@ class SearchUsersEndpointRequest: SearchEndpointRequest {
     }
     
     override func processResponseJSON(_ json: JSON) {
-        processedResponseValue = json["data"]["users"].map{ userJSON in User(json: userJSON.1) }
+        var users: [User] = []
+        json["data"]["users"].forEach { (s,json) in
```

Comment:
why not just use:
for (s,json) in json["data"]["users"] {
}

Looks cleaner IMO. Swift has pattern matching o0o0o0o

Reply:
Ahh yea sure words for me